app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -109,8 +112,10 @@ private class HomeTileAdapter(val onUrlEnteredListener: OnUrlEnteredListener, ho
     override fun onBindViewHolder(holder: TileViewHolder, position: Int) = with (holder) {
         val item = tiles[position]
         titleView.setText(item.title)
-        val bmImg = itemView.context.assets.open(item.imagePath).use { BitmapFactory.decodeStream(it) }
-        iconView.setImageBitmap(bmImg)
+        if (!item.imagePath.isNullOrEmpty()) {
+            val bmImg = itemView.context.assets.open(item.imagePath).use { BitmapFactory.decodeStream(it) }
+            iconView.setImageBitmap(bmImg)
+        }
```

Comment:
nit: should we have a placeholder if it's null/empty?

Reply:
I'd assume that the "get screenshot" code will provide something if it's empty? but if it doesn't we can add it later.